access-control-app/docs/site.md

```diff
@@ -0,0 +1,45 @@
+## Site Routes &amp; Web Resources Documentation
+
+See also the [API Documentation](api.html).
+
+### Table of Contents
+
+  * [Routes](#routes)
+  * [Static Assets and Content](#static-assets-and-content)
+  * [Redirects](#redirects)
+
+### <a name="routes"></a> Routes
+
+The CMR Search Site defines the following application routes, relative to the base CMR Search URL. These resources at the URLs are generated dynamically using page templates (cached).
```

Comment:
Looks like this is copied from CMR search. Shouldn't we change CMR Search to CMR Access Control through out the document?